bin/cron/update_project_count

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+#
+#
+
+require 'sequel'
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/only_one'
+require 'cdo/properties'
+require src_dir 'database'
+require 'cdo/honeybadger'
+
+# Production database has a global max query execution timeout setting.  This 20 minute setting can be used
+# to override the timeout for a specific session or query.
+MAX_EXECUTION_TIME = 1_200_000
+MAX_EXECUTION_TIME_SEC = MAX_EXECUTION_TIME / 1000
+
+# Connection to read from Pegasus reporting database.
+PEGASUS_REPORTING_DB_READER = sequel_connect(
+  CDO.pegasus_reporting_db_reader,
+  CDO.pegasus_reporting_db_reader,
+  query_timeout: MAX_EXECUTION_TIME_SEC
+)
+
+# This default number of lines of code is from 2019-1-21.
+DEFAULT_NUMBER_OF_PROJECTS = 35_000_000
+
+def main
+  # Get the current amount
+  project_count = PEGASUS_REPORTING_DB_READER["SELECT COUNT(standalone) FROM storage_apps WHERE standalone = 1"].first.first[1]
```

Comment:
I tested this in `pegasus-console` on my development environment and it seems to work.  I think it's preferable to use the Sequel query methods to using raw SQL.
```suggestion
  project_count = PEGASUS_REPORTING_DB_READER[:storage_apps].where(standalone: true).count
```